eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -400,6 +400,11 @@ the main ServiceBusClientBuilder. -->
   <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" files="com.azure.spring.messaging.container.AbstractListenerContainer.java"/>
   <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" files="com.azure.spring.cloud.autoconfigure.keyvault.environment.KeyVaultOperation.java"/>
 
+  <suppress checks="MethodName" files="com.azure.spring.data.gremlin.common.GremlinConfig.java"/>
+
+  <suppress checks="MethodName" files="com.azure.spring.integration.core.api.(CheckpointConfig|BatchConsumerConfig).java"/>
+  <suppress checks="MethodName" files="com.azure.spring.integration.servicebus.ServiceBusClientConfig.java"/>
```

Comment:
why is there spring file in your PR?

Reply:
It's from a merging conflict resolution. Seems these are deleted in a previous commit and I added them back.. Will delete these.

Reply:
Removed.